modules/ROOT/pages/_partials/installing-mobile-services.adoc

```diff
@@ -70,16 +85,9 @@ Cluster IP (Defaults to < Network IP Address >)
 Wildcard DNS Host (Defaults to nip.io):
 ----
 +
-The first two values configure the location of the APBs used for the service-catalog in the cluster you are creating.
```

Comment:
Uncluttered the procedure - moved this secondary info under "additional-resources"